src/dashboard/src/components/administration/views.py

```diff
@@ -310,91 +390,53 @@ def _usage_get_directory_used_bytes(path):
         return 0
 
 
-def clear_context(request, dir_id):
-    """
-    Confirmation context for emptying a directory
+def _usage_clear_context(request, dir_id):
+    """Confirmation context for emptying a directory from _get_shared_dirs.
 
-    :param dir_id: Key for the directory in _usage_dirs
+    :param dir_id: Key for the directory in _get_shared_dirs.
     """
-    usage_dirs = _usage_dirs(False)
-    prompt = "Clear " + usage_dirs[dir_id]["description"] + "?"
+    usage_dirs = _get_shared_dirs()
+    dir_info = usage_dirs.get(dir_id, None)
+    if not dir_info:
+        raise Http404
+
+    prompt = _("Clear %(dir)s?") % {"dir": dir_info["description"]}
```

Comment:
Yes, `_()` is usually [the translations wrapper](https://github.com/artefactual/archivematica/blob/stable/1.9.x/src/dashboard/src/components/administration/views.py#L33). By using this syntax it gives a little more context to the translator about the string and allows to change the placeholder location easily. This may not be the best example, but look at [this one from the docs](https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#standard-translation) (at the bottom of the "Standard translation" section).